kernel/common/binding.rb

```diff
@@ -54,4 +54,12 @@ def eval(expr, filename=nil, lineno=nil)
 
     Kernel.eval(expr, self, filename, lineno)
   end
+
+  def local_variables
+    eval('local_variables')
+  end
+
+  def receiver
+    @self
```

Comment:
To make it more obvious for others in the future, it seems like the `@self` ivar could be renamed to `@receiver`, so that `Binding#receiver` would just be an `attr_reader`.